src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig

```diff
@@ -527,3 +527,11 @@
     padding: 5px 0;
     margin-right: 10px;
 }
+
+/***** Media query print: Do not print the Toolbar. *****/
+@media print {
+    .sf-toolbar, .sf-error-toolbar {
+        display: none;
+        visibility: hidden;
```

Comment:
It was in the original CSS rule ... but you are right and this is no longer needed (Bootstrap v4 doesn't use that either. See https://github.com/twbs/bootstrap/blob/v4-dev/scss/_print.scss)